docs/core/tutorials/using-on-macos.md

```diff
@@ -29,7 +24,7 @@ it points out the differences for the Windows platform.
 ## Prerequisites
 
 Before starting, you'll need to install the [.NET Core SDK](https://www.microsoft.com/net/core),
-currently in a preview release. The .NET Core SDK includes the latest release
+currently in RC4 pre-release. The .NET Core SDK includes the latest release
```

Comment:
Let's just leave as .NET Core SDK right now with no version indication and we can fix that later when we know more @BillWagner